packages/react-examples/src/react/Rating/Rating.Basic.Example.tsx

```diff
@@ -13,47 +13,36 @@ export const RatingBasicExample: React.FunctionComponent = () => {
         max={5}
         size={RatingSize.Large}
         defaultRating={1}
-        getAriaLabel={getRatingComponentAriaLabel('Large stars')}
-        ariaLabelFormat="Select {0} of {1} stars"
+        ariaLabel="Large stars"
+        ariaLabelFormat="{0} of {1} stars"
```

Comment:
Is the idea of removing "select" from the examples that it should be added automatically now that the stars' role has been changed to `radio`?

What is read if someone was already passing a string including "select"?

Reply:
"Select" actually should never have been part of the label at all, and anyone currently passing a string with "select" is already making a less-than-ideal choice 😅. The changes in this PR don't make the wording any worse than it already was.

As a rule of thumb, words that indicate states, roles, and specific actions like "select/selected", "button", "on/off", or "press enter/space" shouldn't be part of a control's accessible name. The idea that activating the control will select its value is already built into the idea of a button or radio, and doesn't need to be re-stated in its name. This case is especially bad, since the word "select" is repeated for every button/radio in a set, adding that much more verbosity.